vortex-array/src/compress.rs

```diff
@@ -1,3 +1,4 @@
+// FIXME(ngates): move this file into the compressor
```

Comment:
this was meant as an interface to the compressor, you need to still load the implementation. You can completely kill this file, modulo the asserts